package.json

```diff
@@ -1,73 +1,96 @@
 {
-  "name": "api",
-  "version": "0.0.1",
-  "description": "",
-  "author": "",
+  "name": "@ae_utbm/api",
+  "version": "1.0.0",
   "private": true,
-  "license": "UNLICENSED",
+  "license": "GPL v3",
+  "type": "commonjs",
   "scripts": {
-    "build": "nest build",
-    "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "start": "nest start",
+    "up": "pnpm update && cd src/exported && pnpm update && cd ../..",
+    "lint": "prettier --write . && eslint \"{src,tests}/**/*.ts\" --fix",
+    "test": "pnpm workflow:init_db && jest --config jest.config.ts",
+    "clean": "node clean.mjs",
+    "db:create": "pnpm mikro-orm schema:fresh --run",
+    "db:migrate": "pnpm mikro-orm migration:up",
+    "db:seed": "pnpm mikro-orm seeder:run",
+    "docker:build": "docker build -t 'ae_utbm/api' .",
```

Comment:
docker command should not be in the package.json file. It is the role of the workflow to generate a docker image.

Reply:
Yup, my bad that's an oversight from me